tket/src/Transformations/include/Transformations/Decomposition.hpp

```diff
@@ -92,8 +92,51 @@ Transform decompose_ZX_to_HQS1();
 // single-qubit gates
 Transform decompose_MolmerSorensen();
 
-// finds all ZZ gates which are in the form of CX and Rz gates and converts
-// into ZZ gates Expects: CX, Rz Produces: ZZ, CX, Rz
+/**
+ * @brief A simple struct to store some two-qubit gate fidelities.
+ *
+ * We currently support CX, ZZMax and ZZPhase.
+ *
+ * In the Python layer, this struct is replaced by keyword arguments.
```

Comment:
I'd delete this. (The C++ layer shouldn't document what the Python layer does.)

Reply:
Done.